### ISSUE TITLE
please support compile in deno-1.9.0, got error after compile

### ISSUE THREAD

**Issue by transtone** [denodrivers/redis#224]
deno-1.9.0

when run with `deno run`, it works fine.

when run after compile, it got error: 
```shell
ReferenceError: deferred is not defined
    at MuxExecutor.exec (file://$deno$/bundle.js:12512:19)
    at RedisImpl.execBulkReply (file://$deno$/bundle.js:13318:43)
    at RedisImpl.hget (file://$deno$/bundle.js:13784:21)
```

`vendor/https/deno.land/std` version is 1.8.2, so  `denodrivers/redis` not suppose `>deno-1.8.2` ?

downgrade to `deno-1.8.2`, it not work. 
downgrade to `deno-1.7.5`, it works.

**Comment by uki00a** [comment]
Hi @transtone! I've added support for deno@v1.9.0 in [deno-redis@v0.22.0](https://github.com/denodrivers/redis/releases/tag/v0.22.0). Thanks for reporting this issue!

**Comment by transtone** [comment]
```shell
ReferenceError: deferred is not defined
    at MuxExecutor.exec (file://$deno$/bundle.js:12639:19)
    at RedisImpl.execBulkReply (file://$deno$/bundle.js:12942:43)
    at RedisImpl.hget (file://$deno$/bundle.js:13408:21)
 ```
the error is still there with https://github.com/denodrivers/redis/releases/tag/v0.22.0

**Comment by uki00a** [comment]
Hmmm, I was able to reproduce it too. :cry:  It seems to be a problem with the `deno compile` command itself. :thinking:

https://github.com/denoland/deno/issues/10218

**Comment by transtone** [comment]
> Hmmm, I was able to reproduce it too. 😢 It seems to be a problem with the `deno compile` command itself. 🤔
> 
> [denoland/deno#10218](https://github.com/denoland/deno/issues/10218)

It solved in `deno-1.9.2`
